_includes/_global/required-apis.md

```diff
@@ -0,0 +1,6 @@
+ * **Admin SDK API:** `admin.googleapis.com`
+ * **Cloud Resource Manager API:** `cloudresourcemanager.googleapis.com`
+ * **Cloud SQL Admin API:** `sqladmin.googleapis.com`
+ * **Cloud SQL API:** `sql-component.googleapis.com`
+ * **Compute Engine API**: `compute-component.googleapis.com`
+ * **Deployment Manager API:** `deploymentmanager.googleapis.com`
```

Comment:
Can we add Andrews explanation of how the required API's are determined?  Andrew and I think it's a good idea.
```
It depends on where quota is calculated. If it is on the caller project then the API needs to be enabled on the Forseti project, if it is on the called project, then it is only needed there.
```